peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -129,10 +147,68 @@ <h5>{% translate 'Related documents' %}</h5>
       </div>
     </div>
 
-    {{ citation_links|json_script:"citation-links" }}
+        {{ citation_links|json_script:"citation-links" }}
 
-    {% block content %}
-      {% include 'peachjam/_document_content.html' with document=document %}
-    {% endblock %}
+        {% block content %}
+          {% include 'peachjam/_document_content.html' with document=document %}
+        {% endblock %}
+      </div>
+      <div class="tab-pane fade" id="history-tab" role="tabpanel" aria-labelledby="history-tab">
+        <div class="card">
+        <div class="card-body">
+          <h4>History of this document</h4>
+
+          {% if timeline_events %}
+            <div class="vertical-timeline">
+              {% for history_item in timeline_events %}
+                <div class="vertical-timeline__item">
+                    <div class="vertical-timeline__item__content">
```

Comment:
@musangowope I think you've been too smart here :) Bootstrap's card component gives us a lot of styling (with visual consistency with the rest of the site) which we can use here and save us implementing custom styles.

@nickmwangemi Please replace the custom styling from `vertical-timeline__item__content` downwards with Bootstrap 5's cards. Something like this:

```html
<div class="card">
  <div class="card-header">
    <h5 class="mb-0">
      {{ history_item.date|parse_string_date|date:"d F Y" }}
      {% if history_item.date == current_object_date %}
        <span class="badge rounded-pill bg-primary">this version</span>
      {% endif %}
    </h5>
  </div>

  <div class="card-body">
    {% for event in history_item.events %}
      ...
    {% endfor %}
  </div>
</div>
```

Then you can remove the unused CSS from the .scss file.

Reply:
Done.